development/git_lfs.rst

```diff
@@ -0,0 +1,156 @@
+########################################################
+Using Git Large File Storage (LFS) for Data Repositories
+########################################################
+
+DM uses Git LFS to manage test datasets within our :doc:`normal Git workflow </development/workflow>`.
```

Comment:
Oh yeah. It's a purposefully broken link because it refers to content that will exist one #4 is merged.